### DISCUSSION TITLE
User customizations or settings

### DISCUSSION THREAD

**Discussion by moltar** [community/community#5484] [Codespaces]
Let's say we need to provide each developer with a sandbox AWS account to deploy to.

How would the Account ID, and secrets be managed per developer in the Codespaces environment?

**Comment by moltar** [comment]
Ok, to answer my own question.

Secrets for persons (GitHub users) can be set via this screen: https://github.com/settings/codespaces

Seems kinda an awkward place though.

Why can't users set secrets, or really any config, per repo?

**Comment by tanmayeekamath** [comment]
Hey @moltar, in addition to user level secrets, you can also set secrets at a repo or org level.
[Repo level secrets](https://docs.github.com/en/codespaces/managing-codespaces-for-your-organization/managing-encrypted-secrets-for-your-repository-and-organization-for-codespaces#adding-secrets-for-a-repository)
[Org level secrets](https://docs.github.com/en/codespaces/managing-codespaces-for-your-organization/managing-encrypted-secrets-for-your-repository-and-organization-for-codespaces#adding-secrets-for-an-organization)

Does this help? :)

**Reply by moltar** [reply]
Thank you for sharing. No, this does not help, as it's the opposite of what I was looking for.

I was looking for a way to provide user specific settings  on per repo basis. The settings could be secret, but could also be not secrets.
